config/staging/jenkins.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/jenkins-master/config.xml b/jenkins-master/config.xml
```

Comment:
Yes, those patch files are important because they contain the specific
configuration for the staging and production systems.
Am 23.06.2015 00:25 schrieb "David Burns" <notifications@github.com>:

> In config/staging/jenkins.patch
> <https://github.com/mozilla/mozmill-ci/pull/596#discussion_r32991138>:
>
> > @@ -1,5 +1,5 @@
> >  diff --git a/jenkins-master/config.xml b/jenkins-master/config.xml
>
> Did you mean to have these .patch files?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/mozmill-ci/pull/596/files#r32991138>.
>